sdk/src/types/block/context_input/commitment.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use derive_more::{Display, From};
+
+use crate::types::block::slot::SlotCommitmentId;
+
+/// A Commitment Context indicates that the input references a commitment to a certain slot.
```

Comment:
Just `Commitment Context` is a bit weird

Reply:
from the TIP: `A Commitment Input is an input that allows to reference a commitment to a certain slot` is it better? I didn't like `input is an input` so I changed it a bit.

Reply:
I would either just say `Commitment` or `Commitment Input`